common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -117,10 +74,62 @@
             renderElements: function () {
                 var languages = this.state.config.transcriptLanguages;
 
+                var langTemplate = [
+                    '<div class="grouped-controls">',
+                        '<button class="control toggle-captions" aria-disabled="false">',
+                            '<span class="icon-fallback-img">',
+                                '<span class="icon fa fa-cc" aria-hidden="true"></span>',
+                                '<span class="sr control-text">',
+                                    gettext('Turn on closed captioning'),
+                                '</span>',
+                            '</span>',
+                        '</button>',
+                        '<button class="control toggle-transcript" aria-disabled="false">',
+                            '<span class="icon-fallback-img">',
+                                '<span class="icon fa fa-quote-left" aria-hidden="true"></span>',
+                                '<span class="sr control-text">',
+                                    gettext('Turn off transcript'),
+                                '</span>',
+                            '</span>',
+                        '</button>',
+                        '<div class="lang menu-container" role="application">',
+                            '<button class="control language-menu" aria-label="',
+                                /* jshint maxlen:300 */
+                                gettext('Language: Press the UP arrow key to enter the language ',
+                                'menu then use UP and DOWN arrow keys to navigate language options. ',
+                                'Press ENTER to change to the selected language.'),
+                                '" aria-disabled="false">',
+                                '<span class="icon-fallback-img">',
+                                    '<span class="icon fa fa-caret-left" aria-hidden="true"></span>',
+                                    '<span class="sr control-text">',
+                                        gettext('Open language menu'),
+                                    '</span>',
+                                '</span>',
+                            '</button>',
+                        '</div>',
+                    '</div>'
+                ].join('');
+
+                var template = [
+                    '<div class="subtitles" role="region" aria-label="',
+                        gettext('Video transcript: activating an item in this group will spool the video ',
```

Comment:
You can't break up JS strings into multiple lines and have it work with gettext. Please use single-line strings and also test that translations are working properly.

See http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#don-t-append-strings-interpolate-values

And
http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#coverage-testing